### ISSUE TITLE
Classification of un-classified reads for between-sample comparison

### ISSUE THREAD

**Issue by tuc289** [DerrickWood/kraken2#448]
Hello, 

First off, I really enjoy using Kraken2 on my dataset and it worked pretty well so far. I really appreciate your effort on the community. 

I am having a quick question on un-classified reads. I am wondering if there is a way to classify those un-classified reads from the first run. The reason why I want to check is that there is a possibility to have a bunch of un-classified reads that are identical between samples so that I am missing a lot of information only due to the absence of reference sequences in the DB. 

In short, I want to know if there is a way that I can identify those unclassified reads between samples so that I can have a better idea of which identical reads were not classified using current DB so that I can include more information in the DB for better classification by re-running the program.

If the question is not clear, please let me know and I apologize for any confusion. 

Thanks,

**Comment by kim-fehl** [comment]
Hello, you can use [KrakenTools](https://github.com/jenniferlu717/KrakenTools) to extract unclassified reads and then cluster those reads using different approaches, for instance, [CD-HIT](http://weizhongli-lab.org/cd-hit/).
